example/lib/peripheral_list/components/peripheral_row_view.dart

```diff
@@ -26,12 +33,19 @@ class PeripheralRowView extends StatelessWidget {
             ),
             trailing: _buildListTileTrailing(),
             dense: true,
+            onTap: () => _handleTap(navigatorBloc),
           ),
         ),
       ),
     );
   }
 
+  void _handleTap(NavigatorBloc navigatorBloc) {
+//    peripheralListBloc.add(PickPeripheral(_peripheral));
+    print('elo');
```

Comment:
it seems to be obsolete

Reply:
Definitely a leftover from development, obsolete code was removed.